pkg/daemon/daemon.go

```diff
@@ -253,9 +264,28 @@ func (dn *Daemon) addProfileConfig(socketPath string, configFile string, nodePro
 	return nil
 }
 
+// Add fifo scheduling if specified in nodeProfile
+func addScheduling(nodeProfile *ptpv1.PtpProfile, cmdLine string) string {
+	if nodeProfile.PtpSchedulingPolicy != nil && *nodeProfile.PtpSchedulingPolicy == "SCHED_FIFO" {
+		if nodeProfile.PtpSchedulingPriority == nil {
+			glog.Errorf("Priority must be set for SCHED_FIFO; using default scheduling.")
+			return cmdLine
```

Comment:
should we start the process if the configuration is not right?

It will be super hard for the user to understand that is not running with the policy

Reply:
The intent is to block this from being configured in the ptp operator, so this check is basically redundant.  A user would need to somehow disable validation once it's in.  In my opinion, logging if this somehow happens is enough.

Reply:
I don't think this is right let's say there is a request to have the option to disable the webhook (it will happen) then for the user the linuxptp will start but he will see performance issues on open bugs on that.

It will take much more time to debug a case to find the chrt was not applied than just not start the ptp4l process with the error why.

remember we print all the ptp4l outputs it will be really hard to find this (maybe the log will get rotated)